src/backend/cgxmm.c

```diff
@@ -476,6 +476,92 @@ code *xmmopass(elem *e,regm_t *pretregs)
     return cat4(cr,cl,cg,co);
 }
 
+/********************************
+ * Generate code for post increment and post decrement.
+ */
+
+code *xmmpost(elem *e,regm_t *pretregs)
+{
+    elem *e1 = e->E1;
+    elem *e2 = e->E2;
+    tym_t ty1 = tybasic(e1->Ety);
+
+    CodeBuilder cdb;
+
+    regm_t retregs;
+    unsigned reg;
+    bool regvar = FALSE;
+    if (config.flags4 & CFG4optimized)
+    {
+        // Be careful of cases like (x = x+x+x). We cannot evaluate in
+        // x if x is in a register.
+        unsigned varreg;
+        regm_t varregm;
+        if (isregvar(e1,&varregm,&varreg) &&    // if lvalue is register variable
+            doinreg(e1->EV.sp.Vsym,e2)          // and we can compute directly into it
+           )
+        {   regvar = TRUE;
```

Comment:
I know, much of that is the legacy of 24 line displays. (This particular piece was cut/pasted from elsewhere.)